### PR TITLE
User info 6

### PR DESCRIPTION
**Pull request by CHOCOLVT**

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sdp/blindwar/SplashScreenActivityTest.kt [hunk 1]
```diff
@@ -37,5 +41,13 @@ class SplashScreenActivityTest : TestCase() {
         // the activity pass to the next one too early, don't have time to run tests
 //        assertHasDrawable(R.id.logo_splash_screen, R.drawable.logo)
 //        assertProgressIsMin(R.id.progressBar)
+        //Placeholder
+        Assert.assertEquals(1, 1)
+    }
+
+    @Test
```
Comment (CHOCOLVT) [inline app/src/androidTest/java/ch/epfl/sdp/blindwar/SplashScreenActivityTest.kt:48]: this test should be deleted...

**Review by kapapp — APPROVED** [review]
Set values in userdatabase should be de-duplicated

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/UserNewInfoActivity.kt [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+                    birthDate,
+                    profilePicture.toString(),
+                    gender,
+                    description
+                ) // TODO : Comment for TESTing -> need to uncomment
+//            AuthUI.getInstance().delete(this) // TODO : uncomment for TESTing
+                startActivity(Intent(this, MainMenuActivity::class.java))
+            } else {
+                FirebaseAuth.getInstance().currentUser?.let {
+                    UserDatabase.setPseudo(it.uid, pseudo)
```
Comment (ghemDD) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/UserNewInfoActivity.kt:140]: Maybe create setUser fun in UserDatabase with the given attributes

**Review by ghemDD — APPROVED** [review]
Good testing 
Lacks documentation at some parts

**Pull request merged**